xbatcher/tests/test_accessors.py

```diff
@@ -40,6 +64,25 @@ def test_batch_accessor_da(sample_ds_3d):
         assert batch_class.equals(batch_acc)
 
 
+@pytest.mark.parametrize(
+    "foo_var",
+    [
+        "foo",  # xr.DataArray
+        ["foo"],  # xr.Dataset
+    ],
+)
+def test_tf_to_tensor(sample_ds_3d, foo_var):
+    tensorflow = pytest.importorskip("tensorflow")
+
+    foo = sample_ds_3d[foo_var]
+    t = foo.tf.to_tensor()
+    assert isinstance(t, tensorflow.Tensor)
```

Comment:
Following the usual `import tensorflow as tf` style.

```suggestion
    tf = pytest.importorskip("tensorflow")

    foo = sample_ds_3d[foo_var]
    t = foo.tf.to_tensor()
    assert isinstance(t, tf.Tensor)
```